Sources/ArcGISToolkit/Components/FeatureFormView/Subviews/Inputs/DateTimeInput.swift

```diff
@@ -56,9 +56,11 @@ struct DateTimeInput: View {
                 isEditing = internalFeatureFormViewModel.focusedElement == element
             }
             .onChange(of: date) {
-                element.updateValue(date)
+                if element.value as? Date != date {
+                    element.updateValue(date)
+                    internalFeatureFormViewModel.evaluateExpressions()
+                }
                 formattedValue = element.formattedValue
-                internalFeatureFormViewModel.evaluateExpressions()
```

Comment:
Again, Yoda condition and if instead of guard:

```swift
guard date != element.value as? Date else { return }
element.updateValue(date)
internalFeatureFormViewModel.evaluateExpressions()
formattedValue = element.formattedValue
```